docs/release-notes/2.32.0.md

```diff
@@ -0,0 +1,113 @@
+# Percona Monitoring and Management 2.32.0
+
+| **Release date:** | Nov 7, 2022                                                                                    |
+| ----------------- | ----------------------------------------------------------------------------------------------- |
+| **Installation:** | [Installing Percona Monitoring and Management](https://www.percona.com/software/pmm/quickstart) |
+
+Percona Monitoring and Management (PMM) is an open source database monitoring, management, and observability solution for MySQL, PostgreSQL, and MongoDB.
+
+!!! caution alert alert-warning "Important"
+    We recommend using the latest version of PMM. This ensures that you have access to the latest PMM features and that your environment runs on the latest version of the underlying components, such as VictoriaMetrics, with all the bug fixes in place.
+
+## Release Highlights
+
+
+### State-of-the-art new Home dashboard
+
+In our constant effort to enhance user experience in PMM and considering your feedback, we have added another feather to our cap by replacing the existing home dashboard with a brand new Home dashboard in PMM 2.32.0.  
+
+This new experimental dashboard will be shipped to the PMM users. This dashboard will be uploaded to the Experimental folder to enable the users to test them in their environment and provide feedback.
+
+A few of the improvements made to the new Home Dashboard have been summed up below:
+
+- Optimized performance to handle a large number of nodes concurrently
+- Effortless anomaly detection to check the health of your nodes from the Home dashboard literally at the click of a button.
+- Colored states on panels for a quick visual representation of the problem areas
+- Improved visualization of the Used Memory Anomaly Panel
+- Added a unit (%) to the CPU Anomalies panel for precision
+- Added and refined tooltips for all the panels on the dashboard
+- Enhanced the High CPU Usage, High Disk Queue, and High Memory Usage panels to make the troubleshooting process effortless and more actionable. Just click the node name on the panel to see the details of the problem.
+- Enabled sparklines on the Disk Read and Write panels to represent and show trends in your data visually.
+ 
+
+
+### Enhanced main menu
+
+- In PMM 2.32.0, you can easily access all dashboards for monitored services due to the contemporized navigation structure. All the dashboards in the services folder that were not accessible from the main menu are now accessible from the main menu via **Other Dashboards**. Also, your custom dashboards can be accessed easily from the main menu via **Other Dashboards**
+
+    For example, to access the dashboards for MYSQL, go to *MYSQL >> Other dashboards*.
+
+- Only monitored services are now included in the main menu, enhancing the user experience and minimizing cognitive load to maximize usability.
+
```

Comment:
Can we add an example here? e.g. If you are only monitoring MySQL service, main menu will present MySQL dashboards instead of listing all dashboards that you're not monitoring.

Reply:
Done